draft-ietf-rats-msg-wrap.md

```diff
@@ -162,7 +162,7 @@ This document reuses the terms defined in {{Section 2 of -senml-cf}}
 
 # Conceptual Message Wrapper Encodings
 
-Two types of RATS Conceptual Message Wrapper (CMW) are specified in this
+Two basic types of RATS Conceptual Message Wrapper (CMW) are specified in this
 document:
```

Comment:
```suggestion
A RATS Conceptual Message Wrapper (CMW) has a tree structure of leaves that contain payload messages associated with their content type.
The two leaf node types are
```

Below the enumeration, I'd say "Intermediate tree nodes are either a "collection" type or a "tunnel" type:" and put the descriptive sentences in their own enumeration as well.

Reply:
OK, thanks.  Done in 092be4e and 6656649